keentools_facebuilder/blender_independent_packages/pykeentools_loader/install.py

```diff
@@ -124,20 +147,55 @@ def _download_with_progress_callback(url, progress_callback,
     return result
 
 
+_MAX_CALLBACK_UPDATES_COUNT = 481
+
+
 def install_from_download(version=None, nightly=False, progress_callback=None,
                           final_callback=None, error_callback=None,
-                          max_callback_updates_count=481):
+                          max_callback_updates_count=_MAX_CALLBACK_UPDATES_COUNT):
+    url = download_core_path(version, nightly)
+
+    def install_process(data):
+        _install_from_stream(data, PartInstallation.CORE)
+
+    _download_and_process(url, install_process, progress_callback, final_callback,
+                          error_callback, max_callback_updates_count, None)
+
+
+def _download_zip(part_installation, timeout, version=None, nightly=False, progress_callback=None,
+                  final_callback=None, error_callback=None,
+                  max_callback_updates_count=_MAX_CALLBACK_UPDATES_COUNT,
+                  wait=False):
+    if wait:
```

Comment:
Remove this! There should be better ways to do the same thing (make user see something changed)

For example
- add timestamp to the error message (e.g. user clicks - timestamp changes) (e.g. 20:23:21 - failed to download)
- at least move wait to calling UI code (still bad)